app/controllers/api/v1/admin/statuses_controller.rb

```diff
@@ -48,4 +55,11 @@ def unsensitive
   def set_status
     @status = Status.find(params[:id])
   end
+
+  def destroy_associated_media!
+    @status.ordered_media_attachments.each do |media|
+      media.destroy
```

Comment:
I believe this only removes the Ruby object that keeps track of the media. In order to actually delete the media from S3/filesystem/wherever it is stored, we need to use the `delete` method from `AttachmentBatch`, similar to how `vacuum_orphaned_records` does.